### PR TITLE
fix: cleanup docs about `deps` and `icon` commands

### PR DESCRIPTION
**Pull request by FabianLars**

### CODE REVIEW

File: docs/development/updating-dependencies.md [hunk 1]
```diff
@@ -33,14 +34,30 @@ npm install vue-cli-plugin-tauri@latest
 
 ### Update Cargo Packages
 
-Go to `src-tauri/Cargo.toml` and change `tauri` to
-`tauri = { version = "%version%" }` where `%version%` is the version number shown above. (You can just use the `MAJOR.MINOR`) version, like `0.9`.
+You can check for outdated packages with [`cargo outdated`] or on the crates.io pages: [tauri] / [tauri-build].
```
Comment (jasikpark) [inline docs/development/updating-dependencies.md:30]: Thank you for fixing up the `cargo outdated` link -- was about to push up a PR to fix that :)

File: docs/guides/icons.md [hunk 2]
```diff
@@ -1,18 +1,54 @@
...
+
+or globally:
+
+```bash
+npm install -g github:tauri-apps/tauricon
+# OR
+yarn add -g github:tauri-apps/tauricon
+```
+
+If you decide to use Tauri as a local package with npm (not yarn), you need to add a custom script to your package.json:
```
Comment (amrbashir) [inline docs/guides/icons.md]:
```suggestion
If you decide to use `tauricon` as a local package with npm (not yarn), you need to add a custom script to your package.json:
```

File: docs/development/updating-dependencies.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ Especially during the alpha and beta phases, we expect you to keep all Tauri dep
 ## Automatic updates
 
 The Tauri JS CLI has a command to install and update all needed dependencies, just run `tauri deps install` or `tauri deps update`.
+**Note**: This command is unavailable in CLI versions starting from `v1.0.0-rc.0`
```
Comment (amrbashir) [inline docs/development/updating-dependencies.md]: Probably remove the section altogether. I don't see us providing that command again.

**Review by amrbashir — CHANGES_REQUESTED** [review]

File: docs/development/updating-dependencies.md (hunk 3 above)
Comment (FabianLars) [reply docs/development/updating-dependencies.md]: That was meant for users that want to upgrade from beta.8

Thinking about it it's probably better if they do it manually too 👍

**Review by amrbashir — APPROVED** [review]

**Pull request merged**